tsfresh/feature_extraction/feature_calculators.py

```diff
@@ -1245,3 +1245,43 @@ def _phi(m):
         return np.sum(np.log(C)) / (N - m + 1.0)
 
     return np.abs(_phi(m) - _phi(m + 1))
+
+@set_property("fctype", "aggregate_with_parameters")
+def max_fixed_point(x, r, m):
+    """
+    Largest fixed point of dynamics estimated from polynomial, which has been fitted to 
+    the deterministic dynamics of Langevin model as described by
+
+        Friedrich et al. (2000): Physics Letters A 271, p. 217-222
+        *Extracting model equations from experimental data*
+
+    For short time-series this method is highly dependent on the parameters.
+
+    :param x: the time series to calculate the feature of
+    :type x: pandas.Series
+    :param m: order of polynom to fit for estimating fixed points of dynamics
+    :type m: int
+    :param r: number of quantils to use for averaging
+    :type r: float
+
+    :return: Largest fixed point of deterministic dynamics
+    :return type: float
+    """
+    
+    df = pd.DataFrame({'signal': x[:-1], 'delta': np.diff(x)})
+    try:
+        df['quantiles']=pd.qcut(df.signal, r)
+    except ValueError:
+        return x.mean()
+    quantiles = df.groupby('quantiles')
+    
+    result = pd.DataFrame({'x_mean': quantiles.signal.mean(),
+                           'y_mean': quantiles.delta.mean()
+    })
+    
+    try:
+        coeff = np.polyfit(result.x_mean, result.y_mean, deg=m)
+    except:
```

Comment:
Dynamic systems have stable and unstable fixed points. Without noise these systems would be driven to one of their stable fixed points and stay there for ever. So, fixed points resemble an equilibrium state. 

Noise pushes the system out of their equilibrium and the system's dynamics try to bring it back. So, by averaging the change of state we are able to model the deterministic dynamics as polynomial. Its roots are the fixed points of the dynamics. We are taking the largest fixed point. In the dissipative soliton example, this is an estimate for the equilibrium velocity of the dissipative soliton.

![driftbif20170113a_active_brownian_motion](https://cloud.githubusercontent.com/assets/1429437/21911377/69835aa8-d985-11e6-8f01-d64e5c874562.png)
![driftbif20170113a_brownian_motion](https://cloud.githubusercontent.com/assets/1429437/21911386/73da0506-d985-11e6-99b0-45e518994afc.png)